pkg/cmd/cli/describe/describer.go

```diff
@@ -91,7 +91,7 @@ type BuildDescriber struct {
 }
 
 // Describe returns the description of a build
-func (d *BuildDescriber) Describe(namespace, name string) (string, error) {
+func (d *BuildDescriber) Describe(namespace, name string, settings kctl.DescriberSettings) (string, error) {
```

Comment:
follow up to actually honor the settings (e.g. hide the events if ShowEvents==false)?